addons/mail/tests/test_mail_tracking_value.py

```diff
@@ -0,0 +1,25 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestMailTracking(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.MailTracking = cls.env["mail.tracking.value"]
+
+    def test_create_tracking_values_no_display_name(self):
+
+        messsage1 = self.env['mail.message'].create({'message_type': 'email', 'body': 'Initial Value'})
+        messsage2 = self.env['mail.message'].create({'message_type': 'email', 'body': 'New Value'})
+        initial_value = self.env['mail.message'] + messsage1
+        new_value = self.env['mail.message'] + messsage1 + messsage2
+        col_name = "message_ids"
+        col_info = {"type": "one2many"}
+
+        record = self.env.ref("base.res_partner_1")
+        values = self.MailTracking._create_tracking_values(
+            initial_value, new_value, col_name, col_info, record
+        )
+
+        self.assertEqual(values["old_value_char"], f"mail.message,{messsage1.id}")
+        self.assertEqual(values["new_value_char"], f"mail.message,{messsage1.id}, mail.message,{messsage2.id}")
```

Comment:
There's already a `test_mail_track_2many` in `test_mail`
You can just add "Child4" in the test (except you don't give it a name) and add the expected value in `assertTracking`

Notably having tests there allows us to have a more realistic flow for the test (going through the whole tracking process with a forged model)

Reply:
@reth-odoo you mean like this? Any suggestion regarding the cli style fail?

Reply:
@CRogos yes that seems fine :)

You can just change the join into an fstring like it suggests, it's easier than fighting it ^^
(I don't think it really matters one way or another in this case)

Reply:
@CRogos Also for the CLA, I'm pretty sure you can just create a CLA commit here alongside the fix one